scenarios/jwt_authentication/README.md

```diff
@@ -4,17 +4,79 @@
 
 This scenario showcases how to authenticate to Azure Event Grid via JWT authentication using MQTT 5. This scenario is identical to `getting_started` in functionality. 
 
-JWT authentication is documented in [Microsoft Entra JWT authentication and Azure RBAC authorization to publish or subscribe MQTT messages](https://learn.microsoft.com/en-us/azure/event-grid/mqtt-client-microsoft-entra-token-and-rbac) from Event Grid documentation.
+JWT authentication is documented in [Microsoft Entra JWT authentication and Azure RBAC authorization to publish or subscribe MQTT messages](https://learn.microsoft.com/azure/event-grid/mqtt-client-microsoft-entra-token-and-rbac) from Event Grid documentation.
 
 To keep the scenario simple, a single client called "sample_client" publishes and subscribes to MQTT messages on topics shown in the table.  
 
 |Client|Role|Operation|Topic/Topic Filter|
 |------|----|---------|------------------|
-|sample_client|publisher|publish|sample/topic1|
-|sample_client|subscriber|subscribe|sample/+|
+|sample_client|publisher|publish|jwt/topic1|
+|sample_client|subscriber|subscribe|jwt/+|
 
 ## Prerequisites
-This sample involves configuring Event Grid per the specifications in [getting_started](../getting_started). If that sample has not already been set up and run, it should be done before moving onto this one.
+This sample involves configuring Event Grid per the specifications in [setup](../../Setup.md).
+
+## Create the Client
+
+We will use the SubjectMatchesAuthenticationName validation scheme for `sample_client` to create the client from the portal or with the script. Note that if this has already been done via [getting_started](../getting_started/README.md), this step can be skipped and you can move onto `Create topic spaces and permission bindings`.
+
+```bash
+# from folder scenarios/jwt_authentication
+source ../../az.env
+
+az resource create --id "$res_id/clients/sample_client" --properties '{
+    "authenticationName": "sample_client",
+    "state": "Enabled",
+    "clientCertificateAuthentication": {
+        "validationScheme": "SubjectMatchesAuthenticationName"
+    },
+    "attributes": {
+        "type": "sample-client"
+    },
+    "description": "This is a test publisher client"
+}'
+```
+
+## Create topic spaces and permission bindings
```

Comment:
For JWT the authorization is driven by IAM, not permission bindings (as there is no client)